enhancements/cvo/object-removal-manifest-annotation.md

```diff
@@ -0,0 +1,142 @@
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Currently CVO managed object removal is handled by jobs.
+Jobs have many immutable components, so they aren't a great match for CVO managed manifests.
+Instead of the current approach manifests will be used by adding a new annotation to a manifest which will be handled by the CVO causing it to delete the in-cluster object rather than creating/reconciling the manifest.
+This will provide a more straightforward way to remove content.
+
+## Motivation
+
+There should be a straightforward way to remove CVO managed objects.
+
+### Goals
+
+* Users will be able to remove any of the currently managed CVO objects by simply modifying the object's exisitng manifest and adding the new delete annotation.
+
+### Non-Goals
+
+* This enhancement does not add any new objects for managment by CVO therefore the new delete annotation only applies to the currently supported manifests and any that may be added in the future.
+
+## Proposal
+
+When the following annotation appears in a CVO supported manifest and is set to "true" the associated object will be removed from the cluster by the CVO:
+
+```
+apiVersion: apps/v1
+...
+metadata:
+...
+  annotations:
+    release.openshift.io/delete: "true" 
+```
+
+### User Stories
+
+#### Story 1
+
+Remove the cluster-autoscaler-operator deployment.
+The existing cluster-autoscaler-operator deployment manifest 0000_50_cluster-autoscaler-operator_07_deployment.yaml is modified to contain the delete annotation:
+```
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-autoscaler-operator
+  namespace: openshift-machine-api
+  labels:
+    k8s-app: cluster-autoscaler-operator
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+    release.openshift.io/delete: "true"
+...
+```
+
+#### Story 2
+
+In release 4.5 two jobs, openshift-service-catalog-controller-manager-remover and openshift-service-catalog-apiserver-remover, were created to remove the Service Catalog.
+For release 4.6 these jobs and all their supporting cluster objects should be removed.
+This enhancement allows a user to accomplish this in a very simple manner by modifying all the manifests associated with these two jobs to contain the delete annotation.
+In this case the manifests to be modified are:
+* 0000_50_cluster-svcat-apiserver-operator_04_roles.yaml
+* 0000_50_cluster-svcat-apiserver-operator_05_serviceaccount.yaml
+* 0000_50_cluster-svcat-controller-manager-operator_06_roles.yaml
+* 0000_50_cluster-svcat-controller-manager-operator_07_serviceaccount.yaml
+* 0000_90_cluster-svcat-apiserver-operator_01_remover_job.yaml
+* 0000_90_cluster-svcat-controller-manager-operator_01_remover_job.yaml
+
+### Implementation Details/Notes/Constraints
+
+CVO will use it's existing logic to discover and identify given manifests and to handle any associated errors.
```

Comment:
nit: "it's" -> "its"

Reply:
Fixed in commit [262765d](https://github.com/openshift/enhancements/pull/419/commits/262765d0e846f11bb32d17fe4841e5b9917d4a82)